components/home/solutions/Testimonials.tsx

```diff
@@ -92,12 +134,25 @@ const Testimonials = () => {
           </h2>
```

Comment:
I think this section background image above should use `object-contain` instead of `object-cover`:
<img width="1436" height="647" alt="image" src="https://github.com/user-attachments/assets/a58194c2-37e9-48f0-a583-6680c230b04e" />

Also, why use absolute positioned image instead of a simpler background-image selector?

Reply:
Good catch, fixed this.

The absolute positioned image is to make use of the Next.js image optimization